pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -260,7 +260,7 @@ public Connection connect(String url, Properties info) throws SQLException {
       thread.start();
       return ct.getResult(timeout);
     } catch (PSQLException ex1) {
-      LOGGER.log(Level.SEVERE, "Connection error: ", ex1);
+      LOGGER.log(Level.WARNING, "Connection error: {0}", ex1.getServerErrorMessage());
```

Comment:
Why do you replace `ex1` with `getServerErrorMessage`?

Reply:
According to the discussion, to remove the stack trace from the logger.